lib/livebook_web/live/hub/edit_live.ex

```diff
@@ -31,9 +30,8 @@ defmodule LivebookWeb.Hub.EditLive do
        hub: hub,
        type: type,
        secrets: Hubs.get_secrets(hub),
-       params: params,
-       env_var_id: params["env_var_id"],
-       secret_name: params["secret_name"]
+       secret_name: params["secret_name"],
+       show_key: params["show-key"] == "true"
```

Comment:
This is leaking concerns specific to each hub to the parent. I think we should only pass three assigns here: `hub`, `type`, `params`. Everything else should be handled on each individual component.